launcher/src/main/java/org/apache/spark/launcher/SparkLauncher.java

```diff
@@ -64,6 +64,10 @@
   /** Configuration key for the number of executor CPU cores. */
   public static final String EXECUTOR_CORES = "spark.executor.cores";
 
+  public static final String PYSPARK_DRIVER_PYTHON = "spark.pyspark.driver.python";
```

Comment:
@vanzin  Thanks for quick review. This is because they are also used in spark-launcher module (`SparkSubmitCommandBuilder.java`) , so I put them in `SparkLauncher.java`.
Will update the `configuration.md`